clients/benchmarks/CMakeLists.txt

```diff
@@ -40,82 +45,49 @@ set( rocblas_benchmark_common
       ../common/rocblas_template_specialization.cpp
     )
 
-add_executable( client client.cpp ${rocblas_benchmark_common} )
-
-
-add_subdirectory ( ./perf_script )
-
-
+add_executable( rocblas-bench client.cpp ${rocblas_benchmark_common} )
 
 if( BUILD_WITH_TENSILE )
-    target_compile_definitions( client PRIVATE BUILD_WITH_TENSILE=1 )
-    message(STATUS, "Build Tensil equal 1")
+    target_compile_definitions( rocblas-bench PRIVATE BUILD_WITH_TENSILE=1 )
```

Comment:
to help people like me can we change this to BUILD_WITH_TENSILE=ON  rather than BUILD_WITH_TENSILE=1